apps/passport-server/test/user/testUserSync.ts

```diff
@@ -1,53 +1,497 @@
 import {
```

Comment:
@ichub I'd be interested to know if there's a reason for this test code to be in a separate file structured as plain functions, rather than a spec.ts file containing test cases.  Seems like an unnecessary layer of indirection to me, but I followed the pattern I found.

Reply:
no reason for this file. other files follow this pattern - eg `testLogin` - because they are meant to be re-used in a lot of tests.